### DISCUSSION TITLE
Managing NEON resources

### DISCUSSION THREAD

**Discussion by wwieder** [ESCOMP/CTSM#2692] [General]
Currently we have scripts scattered about that manage different aspects of NEON workflows, visualizations, and tutorials.  I don't suggest these be included in the CTSM repository, but maybe we should consider a way to organize / keep track of them?  Today @ekluzek mentioned these could be checked out as optional externals, but I'm not sure what this would entail?

Here's a list of what's available:
- [NCAR-NEON visualization-website](https://github.com/NCAR/NEON-visualization) and associated[ github pages](https://ncar.github.io/NEON-visualization/) for users
- Confusingly, users actually get started with docker using these [NCAR-NEON-books](https://github.com/NCAR/ncar-neon-books).
- [CTSM tutorial repository](https://github.com/NCAR/CTSM-Tutorial-2022) and[ pages](https://ncar.github.io/CTSM-Tutorial-2022/README.html).
- [Plots](https://github.com/wwieder/neon_scripts) used in the paper, forked from @negin513 repository, and needs to be merged under a NCAR hosted repo.
- bokeh plots from @negin513, which are not yet in a NCAR or NEON repo.
- @TeaganKing is starting to make a new set of tools to bring in alternative precipitation data from #1904  
- Finally, there's this [external repo from NEON](https://quay.io/repository/ddurden/ncar-neon) on generating DATM data for the system.

Suggestions about how to facilitate / organize all of this are very welcome.

**Comment by ekluzek** [comment]
Moving to a discussion
